lib/usdUfe/ufe/Utils.cpp

```diff
@@ -130,6 +130,18 @@ UsdUfe::Transform3dMatrixOpNameFn  gTransform3dMatrixOpNameFn = nullptr;
 UsdUfe::DisplayMessageFn gDisplayMessageFn[static_cast<int>(UsdUfe::MessageType::nbTypes)]
     = { nullptr };
 
+inline std::string joinStrings(const std::vector<std::string>& items)
+{
+    std::string result;
+    for (const auto& item : items) {
+        if (!result.empty()) {
+            result += ", ";
+        }
+        result += item;
+    }
+    return result;
+}
```

Comment:
Could you just use [TfStringJoin](https://openusd.org/docs/api/group__group__tf___string.html#ga2693efcb60908bf5b07caa0786ec000a)?